reconcile/automated_actions/config/integration.py

```diff
@@ -158,8 +158,13 @@ def compile_roles(
                             "name": arg.name_pattern,
                         })
                     case _:
```

Comment:
shouldn't this be updated to `case AutomatedActionArgumentActionListV1:` and keep the default unsupported `case _:`?